is_core/auth_token/middleware.py

```diff
@@ -33,10 +33,12 @@ def process_response(self, request, response):
         # Save the session data and refresh the client cookie.
         # Skip session save for 500 responses, refs #3881.
         if response.status_code != 500 and hasattr(request, 'token') and request.token.is_active:
-            if not request.token.expiration:
```

Comment:
Nějak to mu nerozumím. Proč když má token expiraci, se tomu nastavuje `max_age, expires = None`?